infra/modules/monitoring/main.tf

```diff
@@ -0,0 +1,124 @@
+# Create CloudWatch alarms for ECS metrics
+
+resource "aws_cloudwatch_metric_alarm" "cpu" {
+  count                     = var.cpu_threshold != "" ? 1 : 0
+  alarm_name                = "${var.ecs_name}-cpu-usage"
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
+  evaluation_periods        = 2
+  metric_name               = "CPUUtilization"
+  namespace                 = "AWS/ECS"
+  period                    = 120
+  statistic                 = "Average"
+  threshold                 = var.cpu_threshold
+  alarm_description         = "This metric monitors ecs cluster cpu utilization"
```

Comment:
the current description sounds like it's describing a metric. i understand that's what the terraform examples do too, but i don't think those are good examples. let's use a description that describes the alarm itself, e.g. "ECS cluster high CPU utilization alarm"

Reply:
I agree, description is misleading.

Reply:
done